pkg/leaderelection/leaderelection.go

```diff
@@ -31,7 +31,7 @@ func GetLeaderElectionConfig(restConfig *rest.Config, enabled bool) configv1.Lea
 				return leaderelection.LeaderElectionSNOConfig(defaultLeaderElection)
 			}
 		} else {
-			klog.Warningf("unable to get cluster infrastructure status, using HA cluster values for leader election: %v", err)
+			glog.Warningf("unable to get cluster infrastructure status, using HA cluster values for leader election: %v", err)
```

Comment:
Same here, klog implementation of logr. I believe all business logic , that is outside operator-sdk generated/managed code uses logr/zap/klog and the domain-related code is using glog.